app.rb

```diff
@@ -0,0 +1,65 @@
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+
+class App
+  attr_reader :books, :classrooms, :students, :teachers
+
+  def initialize
+    @students = []
+    @books = []
+    @teachers = []
+    @rentals = []
+    @classrooms = []
+  end
+
+  def list_people
+    @students.each { |s| puts "[Student] ID: #{s.id}, Name: #{s.name}" }
+    @teachers.each { |t| puts "[Teacher] ID: #{t.id}, Name: #{t.name}" }
+  end
+
+  def list_books
+    @books.each { |b| puts "[Book] Title: #{b.title}, Author: #{b.author}" }
+  end
+
+  def list_classrooms
+    @classrooms.each { |c| puts "[Classroom] Label: #{c.label}" }
+  end
+
+  def create_student(age, name, classroom)
+    student = Student.new(age, name, classroom)
+    @students << student
+    student
+  end
+
+  def create_teacher(age, name, specialization)
+    teacher = Teacher.new(age, name, specialization)
+    @teachers << teacher
+    teacher
+  end
+
+  def create_book(title, author)
+    book = Book.new(title: title, author: author)
+    @books << book
+    book
+  end
+
+  def create_rental(person, book)
+    rental = Rental.new(person: person, book: book)
+    @rentals << rental
+    rental
+  end
```

Comment:
- I have this error each time I try to create a Rental, please fix this 🙏 
<img width="729" alt="heh5h5" src="https://user-images.githubusercontent.com/73607512/209143593-901c8233-9adf-485a-98f6-c63063d632cc.PNG">